File: problemsets/Codeforces/Python/A708.py
"""
 *
 * Author:  Juarez Paulino(coderemite)
 * Email: juarez.paulino@gmail.com
 *
 """
a=[*input()]
n=len(a)
i=0
while i<n and a[i]=='a':i+=1
if i==n:a[-1]='z'
while i<n and a[i]!='a':a[i]=chr(ord(a[i])-1);i+=1
print(''.join(a))

File: problemsets/Codeforces/C++/A708.cpp
/**
 *
 * Author:  Juarez Paulino(coderemite)
 * Email: juarez.paulino@gmail.com
 *
 */
#include <bits/stdc++.h>

using namespace std;

#define eb emplace_back
#define fi first
#define se second
typedef pair<int,int> pii;

#define sqr(x) (x)*(x)

int main() {
	ios::sync_with_stdio(0);
	cin.tie(0);
	string a; cin >> a;
	int k = 0;
	while (k < a.size() && a[k] == 'a') k++;
	if (k == a.size()) { a.back() = 'z', cout << a; return 0; }
	while (k < a.size() && a[k]!='a') a[k++]-=1;
	cout << a;
}